website/src/documentation/transforms/java/aggregation/groupbykey.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: section
+title: "GroupByKey"
+permalink: /documentation/transforms/java/aggregation/groupbykey/
+section_menu: section-menu/documentation.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# GroupByKey
+<table align="left">
+    <a target="_blank" class="button"
+        href="https://beam.apache.org/releases/javadoc/current/index.html?org/apache/beam/sdk/transforms/GroupByKey.html">
+      <img src="https://beam.apache.org/images/logos/sdks/java.png" width="20px" height="20px"
+           alt="Javadoc" />
+     Javadoc
+    </a>
+</table>
+<br>
+Takes a keyed collection of elements and produces a collection where
+each element consists of a key and all values associated with that key.
+
+Normally, the result of a `GroupByKey` contains one element for each key
```

Comment:
This sentence might be confusing. Even with windowing and triggering, each time the output is still a (k, Iterable) pair. I would just remove this sentence, and update the one above to "... consists of a key and a `Iterable` of all values associated with that key."